src/App.tsx

```diff
@@ -1,9 +1,18 @@
+import { useState } from 'react';
+
 import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```
react wiill infer type from emtpy string passed as default argument